misc/dbt-materialize/dbt/include/materialize/macros/adapters.sql

```diff
@@ -264,16 +264,6 @@
   comment on column {{ relation }}.{{ adapter.quote(column_name) if quote else column_name }} is {{ escaped_comment }};
 {% endmacro %}
 
-{% macro materialize__alter_relation_comment(relation, comment) -%}
```

Comment:
@bobbyiliev, we should be able to remove this custom code now!

Reply:
Ah nice! Thanks for catching that Marta!